test/common/ops/feed.js

```diff
@@ -89,6 +89,18 @@ describe('shared.ops.feed', () => {
       }
     });
 
+    it('does not allow feeding of wacky pets', done => {
+      user.items.pets['Wolf-Veggie'] = 5;
+      user.items.food.Meat = 1;
+      try {
+        feed(user, { params: { pet: 'Wolf-Vegie', food: 'Meat' } });
```

Comment:
looks like a `g` got lost here :) `Wolf-Vegie` vs `Wolf-Veggie`

Reply:
Well that was embarrassing! Thanks for catching that :)

Reply:
no worries :)